docs/design/Chem Integration.md

```diff
@@ -0,0 +1,69 @@
+# Loose binary identification with Upgrade Assistant
+
+It is not uncommon in large code bases to have a collection of dependencies that have been persisted as only the `.dll` files with no known provenance. These files are considered "loose binaries". This makes it challenging to identify what a binary is while upgrading where an updated version is often needed to support .NET Standard/Core/5. In order to solve this, this document proposes integration with a data source that can help match these loose assemblies to known NuGet packages.
```

Comment:
wasn't clear how long this document will live but I think it's better to introduce a term like *loose binaries* with some prominence that we keep using whenever we reference the word as one of the nouns/topics distinct to our project's ubiquitous language. If this doc lives on I recommend replacing quotes with italic emphasis for a subtle but *distinct* highlight.

It might be helpful to start a glossary at some point.

Reply:
Add this to the glossary!